hive/src/itest/java/org/elasticsearch/hadoop/integration/hive/AbstractHiveSaveTest.java

```diff
@@ -696,15 +695,27 @@ public void testMappingExclude() throws Exception {
         System.out.println(server.execute(selectTest));
         System.out.println(server.execute(insert));
 
-        String string = RestUtils.get("hive-fieldexclude/data/1");
+        String docEndpoint = resource("hive-fieldexclude", "data");
+        if (TestUtils.isTypelessVersion(targetVersion)) {
+            docEndpoint = docEndpoint + "/_doc";
+        }
```

Comment:
Is this right ?

`hive-fieldexclude/data/_doc` ?  or should it be conditionally only `hive-fieldexclude/_doc` ?